design/MonitoringModeAPI.md

```diff
@@ -166,6 +166,88 @@ see [Create Experiment](/design/CreateExperiment.md)
 
 </details>
 
+**Request with `experiment_type` field**
```

Comment:
As we add new features to the API, the effort should be to keep the documentation as simple as possible. This means that we need to have one example for the API that is the one that should be commonly used and everything else can be added as notes for the API. I now see 3 examples, one without `experiment_type`, one with and has `container` and another for `namespace`. I would suggest to have one example for `container` and mention in the notes that it is optional and valid values are `container` and `namespace`. This can be combined with any other notes for this API